submissions/hadizaSaadu/HTML/E-commerce.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>product-page</title>
+  </head>
+  <body>
+    <h1>DEEZAH'S CLOSET</h1>
+    <h3>
+      <i>Home of modest wears . . .</i>
+    </h3>
+
+    <a href="#">HOME</a>
+
+    <a href="#">ABOUT US</a>
+    <a href="#">CONTACT</a>
+    <a href="#">FAQ</a>
+
+    <!--THIS SECTION IS FOR PRODUCTS-->
+    <section>
+      <h3>PRODUCTS SECTION</h3>
+      <table>
+        <tr>
+          <td>
+            <img
```

Comment:
This should take a single line, and remove the `,` here too

Reply:
done